### PR TITLE
monitoring: bare bones prometheus config and rules.

### PR DESCRIPTION
**Pull request by mberhault**
This is meant to be used in the docs and is a working config/rules for a
local node (with a single line to change for secure/insecure).

Work torwards https://github.com/cockroachdb/docs/issues/469

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/12288)

### CODE REVIEW

File: monitoring/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    # Secure mode:
+    # scheme: 'https'
+
+    tls_config:
+      insecure_skip_verify: true
+
+    static_configs:
+    - targets: ['localhost:8080']
+      labels:
+        cluster: 'cockroach-test'
```
Comment (jseldess) [inline monitoring/prometheus.yml]: Let's maybe rename this to something like 'my-cockroachdb-cluster"?

**Review by jseldess — APPROVED** [review]
LGTM, once you add the top-level grafana dashboards, as discussed offline.

**Comment by mberhault** [comment]
Fixed typo and added grafana dashboards. We may want to give a bit of info about each:
* runtime: node status: uptime, memory, cpu, ...
* storage: storage availability
* sql: sql queries/transactions
* replicas: replica information and replica operations

**Comment by jseldess** [comment]
Thanks, @mberhault. Still LGTM.

**Pull request merged**